mail-client/alpine/files/2.21.1-nopam.patch

```diff
@@ -0,0 +1,27 @@
+--- ./configure.ac	2019-11-04 20:09:50.094172283 +0100
++++ ./configure.ac	2019-11-04 20:59:33.775007247 +0100
+@@ -257,6 +257,7 @@
+   alpine_c_client_gccoptlevel="-O0"
+   AC_MSG_RESULT([no])
+ fi
++AC_ARG_WITH([pam], AS_HELP_STRING([--without-foo], [Build without PAM support]))
```

Comment:
`--without-foo` heh :) not that it matters much for Gentoo users, but if someone uses this for themself, etc...

Reply:
Done